rate_limiter/src/lib.rs

```diff
@@ -45,11 +45,8 @@
 //! needs to be called by the user on every event on the rate limiter's `AsRawFd` FD.
 //!
 
-extern crate serde;
```

Comment:
You also need to remove the dependency from `rate_limiter/Cargo.toml`. Afterwards you also need to run `cargo build` so that `Cargo.lock` gets updated. Add the Cargo.lock and Cargo.toml files to this commit as well.

Reply:
Ok, I will do this immediately